### ISSUE TITLE
Gitter mark/popup precedence

### ISSUE THREAD

**Issue by ShrykeWindgrace** [SublimeHaskell/SublimeHaskell#341]
Hi!

I am having troubles with gutter marks and popups precedence. As you can see in the screenshot, the mark comes from GitGutter, but the popup comes from SublimeHaskell:
![hover](https://user-images.githubusercontent.com/6421233/27856260-db5844ca-616d-11e7-9c9e-80ba34b6f2ee.png)

I found the "protected regions" setting for GitGutter, which allows me to prefer gutter marks from other plugins, but in order to use it, I need the name of the region used by SublimeHaskell for its gutter marks. I did not find this name by grepping the source code.

I'll be glad to hear about all suggestions on how to fix mark/popup precedence. Or the name of the gutter region=)

Cheers

**Comment by derekchiang** [comment]
Running into the same issue here.  @ShrykeWindgrace did you ever figure out how to resolve this?

**Comment by ShrykeWindgrace** [comment]
@derekchiang No, I did not really look into it. I switched to other IDEs for Haskell where I can, and disabled GitGutter on machines where I can not. Anyway, the gutter in ST3 is overloaded with info. GitGutter, SublimeHaskell, and BracketHighlighter, all try to show me something in the same space, we definitely need more than one gutter in ST3.
